ci/gdk_build.template.steps.yaml

```diff
@@ -55,14 +55,14 @@ env:
 
 steps:
   - <<: *BUILDKITE_AGENT_PLACEHOLDER
-    label: "build-${ENGINE_COMMIT_HASH}-${BUILD_PLATFORM}-${BUILD_TARGET}-${BUILD_STATE}"
+    label: "build-${ENGINE_COMMIT_HASH}-${BUILD_PLATFORM}-${BUILD_TARGET}-${BUILD_STATE}-${TEST_CONFIG}"
     command: "${BUILD_COMMAND}"
     artifact_paths:
       - "../UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
-      - "ci/FastTestResults/*"
-      - "ci/VanillaTestResults/*"
     env:
+      BUILD_ALL_CONFIGURATIONS: "${BUILD_ALL_CONFIGURATIONS}"
       ENGINE_COMMIT_HASH: "${ENGINE_COMMIT_HASH}"
       BUILD_PLATFORM: "${BUILD_PLATFORM}"
       BUILD_TARGET: "${BUILD_TARGET}"
       BUILD_STATE: "${BUILD_STATE}"
+      TEST_CONFIG: "${TEST_CONFIG}"
```

Comment:
Do we need to pass these environment variables? Wouldn't they be present on any BK agent?

Reply:
nevermind, these variables are defined while running the pipeline